assets/stylesheets/common/chat-transcript.scss

```diff
@@ -0,0 +1,46 @@
+.discourse-chat-transcript {
```

Comment:
Did you get a chance to view this in mobile?

Reply:
No, will make sure it works well there and fix if not

Reply:
@nattsw I think this is fine on mobile:

![image](https://user-images.githubusercontent.com/920448/152445154-1cce968b-f72a-4a3e-aa8d-dfa372111b1c.png)